components/QueryIdSelect.tsx

```diff
@@ -16,6 +16,7 @@ const QueryIdSelect = (props: Props) => {
       >
         <option value="">default</option>
         <option value="languages">languages</option>
+        <option value="exact-title-match">exact-title-match</option>
```

Comment:
is it possible to dynamically pull all the options from the `queries/` directory here instead of adding each one manually?

Reply:
It definitely is, but given we only use it in one place, I didn't bother.